### PR TITLE
First clean-up

### PR DESCRIPTION
**Pull request by bakura10**
Previous PR was too massive, I'm going to split it up in multiple ones so it's easier for you.

This one is mostly removing unused interface, removing the dependency to Doctrine\ORM (Common is sufficient), and renaming test to tests folder for homogeneity with ZF2.

### CODE REVIEW

**Comment by bakura10** [comment]
Possible to merge this ? I'll have some free time soon, I'd like to work on this module ;-).

**Comment by spiffyjr** [comment]
Still at ZendCon so I'm a bit slow.

File: src/SpiffyAuthorize/Guard/GuardInterface.php [hunk 1]
```diff
@@ -2,8 +2,32 @@
...
 use Zend\EventManager\ListenerAggregateInterface;
 
+/**
+ * Interface that each guard must implement
+ *
+ * A guard is a simple protection layer that occurs between the dispatch and response
+ * of your application. A guard allows to simply protect a controller, a route or a
+ * route hierarchy (eg. all routes that are child routes of "admin" route), BUT should
+ * not be used as the only way to protect your application. Your service layer should
+ * also be protected
```
Comment (spiffyjr) [inline src/SpiffyAuthorize/Guard/GuardInterface.php]: Add a period to the end of this last sentence.

**Comment by spiffyjr** [comment]
Remove composer.phar as well.

**Comment by bakura10** [comment]
Done :).

**Comment by spiffyjr** [comment]
You still have to git rm composer.phar! ;)

**Comment by bakura10** [comment]
Haha I think it's time for me to have a global composer.phar file rather than per project I suppose... :I

Envoyé de mon iPhone

**Comment by bakura10** [comment]
Done :)

**Comment by spiffyjr** [comment]
Looks like a manual merge. I'll try to do this tomorrow.

**Comment by bakura10** [comment]
Any news? :)

**Comment by spiffyjr** [comment]
Sorry, got pretty busy. You done committing to this?

**Comment by spiffyjr** [comment]
Also, can you rebase this with master?

**Comment by bakura10** [comment]
Yes I'm done with this one. I'm going to debase when I got home :)

Envoyé de mon iPhone

**Comment by bakura10** [comment]
I screw up eveyrhting. I'll make a new PR.

**Pull request closed without merging**